src/main/kotlin/Main.kt

```diff
@@ -11,4 +14,31 @@ fun problem1(intArray: Array<Int>) : Int {
             sum += index
     }
     return sum
+}
+
+// Write a method that takes an array of consecutive (increasing) letters as input and that returns the missing letter in the array.
+// You will always get a valid array. And it will be always exactly one letter be missing. The length of the array will always be at least 2.
+// The array will always contain letters in only one case.(Use the English alphabet with 26 letters!)
+fun problem2(charArray: Array<Char>) : String{
+    if(charArray.size < 2)
+        return "Dude, your array is too short try something longer :)"
+    var upperCaseOn = true
+    if(charArray[0].code in 97..122)
+        upperCaseOn = false
+    else if(charArray[0].code in 65..90)
+        upperCaseOn = true
```

Comment:
Can you refactor this using Kotlin's switch case? 
https://kotlinlang.org/docs/control-flow.html#when-expression

Reply:
Done it, it will be updated in next commit as an update